app/controllers/WrongWithMovementController.scala

```diff
@@ -43,17 +43,91 @@ class WrongWithMovementController @Inject()(
                                              val controllerComponents: MessagesControllerComponents,
                                              view: WrongWithMovementView
-                                     ) extends BaseNavigationController with AuthActionHelper {
+                                           ) extends BaseNavigationController with AuthActionHelper {
 
   def loadWrongWithMovement(ern: String, arc: String, mode: Mode): Action[AnyContent] =
     onPageLoad(WrongWithMovementPage, ern, arc, routes.WrongWithMovementController.submitWrongWithMovement(ern, arc, mode))
-  def submitWrongWithMovement(ern: String, arc: String, mode: Mode): Action[AnyContent] =
-    onSubmit(WrongWithMovementPage, ern, arc, mode, routes.WrongWithMovementController.submitWrongWithMovement(ern, arc, mode))
 
-  def loadwrongWithItem(ern: String, arc: String, idx: Int, mode: Mode): Action[AnyContent] =
-    onPageLoad(WrongWithItemPage(idx), ern, arc, routes.WrongWithMovementController.submitwrongWithItem(ern, arc, idx, mode))
-  def submitwrongWithItem(ern: String, arc: String, idx: Int, mode: Mode): Action[AnyContent] =
-    onSubmit(WrongWithItemPage(idx), ern, arc, mode, routes.WrongWithMovementController.submitwrongWithItem(ern, arc, idx, mode))
+  def submitWrongWithMovement(ern: String, arc: String, mode: Mode): Action[AnyContent] = {
+    authorisedDataRequestAsync(ern, arc) { implicit request =>
+      formProvider(WrongWithMovementPage).bindFromRequest().fold(
+        formWithErrors =>
+          Future.successful(BadRequest(view(
+            WrongWithMovementPage,
+            formWithErrors,
+            routes.WrongWithMovementController.submitWrongWithMovement(ern, arc, mode)
+          ))),
+        (values: Set[WrongWithMovement]) => {
+
+          val allOptionsNotChecked: Seq[WrongWithMovement] = WrongWithMovement.values.filterNot(values.contains)
+
+          val newUserAnswers = allOptionsNotChecked.foldLeft(request.userAnswers) {
+            case (answers, WrongWithMovement.Shortage) =>
+              answers
+                .remove(AddShortageInformationPage)
+                .remove(ShortageInformationPage)
+            case (answers, WrongWithMovement.Excess) =>
+              answers
+                .remove(AddExcessInformationPage)
+                .remove(ExcessInformationPage)
+            case (answers, WrongWithMovement.Damaged) =>
+              answers
+                .remove(AddDamageInformationPage)
+                .remove(DamageInformationPage)
+            case (answers, WrongWithMovement.BrokenSeals) =>
+              answers
+                .remove(AddSealsInformationPage)
+                .remove(SealsInformationPage)
+            case (answers, WrongWithMovement.Other) =>
+              answers
+                .remove(OtherInformationPage)
+            case (answers, _) => answers
+          }
+
+          saveAndRedirect(WrongWithMovementPage, values, newUserAnswers, mode)
+        }
+      )
+    }
```

Comment:
Should only do this if the set of selected checkboxes is different to the previous set of selected checkboxes. The back and then forward without changing scenario again.